x-pack/plugins/ml/public/application/supplied_configurations/supplied_configurations_flyout/overview_tab_content.tsx

```diff
@@ -0,0 +1,352 @@
+                    <EuiDescriptionListDescription>
+                      <EuiCodeBlock
+                        language="json"
+                        isCopyable
+                        overflowHeight="500px"
+                        data-test-subj="mlPreconfigJobsQueryBlock"
+                      >
+                        {JSON.stringify(module.query, null, 2)}
+                      </EuiCodeBlock>
+                      <EuiSpacer size="s" />
+                    </EuiDescriptionListDescription>
+                  </EuiDescriptionList>
+                </EuiFlexItem>
+                {module.query !== undefined ? (
+                  <EuiFlexItem grow={false}>
+                    <EuiFlexGroup gutterSize="m" alignItems="center" justifyContent="flexEnd">
+                      <EuiFlexItem grow={false}>
+                        <EuiText size={'xs'}>
+                          <FormattedMessage
+                            id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.dataRecognizerHelpButtonLabel"
+                            defaultMessage="Check indices to see which can be used to run jobs in this module?"
+                          />
+                        </EuiText>
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={false}>
+                        <span>
+                          <EuiButton
+                            fill
+                            isDisabled={runningDataRecognizer}
+                            isLoading={runningDataRecognizer}
+                            color={'primary'}
+                            onClick={runDataRecongizer}
+                            size="s"
+                          >
+                            <FormattedMessage
+                              id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.runDataRecognizerButtonLabel"
+                              defaultMessage="Run data recognizer"
+                            />
+                          </EuiButton>
+                        </span>
+                      </EuiFlexItem>
+                    </EuiFlexGroup>
+                  </EuiFlexItem>
+                ) : null}
+                {matchingDataViews.length ? (
+                  <EuiFlexItem grow={false}>
+                    <DataViewsTable matchingDataViews={matchingDataViews} moduleId={module.id} />
+                  </EuiFlexItem>
+                ) : null}
+              </EuiFlexGroup>
+            ) : (
+              <EuiEmptyPrompt
+                css={{ margin: '10px' }}
+                color="primary"
+                iconType={(isLogoObject(module.logo) ? module.logo?.icon : module.logo) as string}
+                title={
+                  <h2>
+                    <FormattedMessage
+                      id="xpack.ml.anomalyDetection.suppliedConfigurationsFlyout.unableToUseModuleTitle"
+                      defaultMessage="You cannot create these jobs here"
```

Comment:
Would it be too much to write the app's name, @alvarezmelissa87?

```suggestion
                      defaultMessage="You cannot create these jobs in Machine Learning"
```